CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- Added handling KeyboardInterrupt during training ([#2134](https://github.com/PyTorchLightning/pytorch-lightning/pull/2134))
```

Comment:
```suggestion
- Added a callback method `on_interrupt` for handling KeyboardInterrupt events ([#2134](https://github.com/PyTorchLightning/pytorch-lightning/pull/2134))
```
the message was misleading, since we already handle KeyboardInterrupt and perform graceful shutdown.